server/models/notifications.js

```diff
@@ -0,0 +1,15 @@
+const notificationnDefinition = (sequelize, DataTypes) => {
+  const Notification = sequelize.define('Notification', {
+    title: DataTypes.STRING,
+    requester: DataTypes.STRING,
+    manager: DataTypes.STRING,
```

Comment:
Why the manager, requester, email, title fields here? I ma not for this kind of entity for a notification. Please change and get something better. we can have just something like from,to (replacing manager and requester), etc . why email? I want you to seat down and do this story nicer. it is lacking problem solving skills.

Reply:
I have made some changes. But the title will be required to define an email.